src/ceph-disk/ceph_disk/main.py

```diff
@@ -4524,9 +4531,35 @@ def list_devices():
     return devices
 
 
+def list_zfs():
+    try:
+        out, err, ret = command(
+            [   'zfs',
+                'list',
+                '-o', 'name,mountpoint'
+            ]
+           )
+    except subprocess.CalledProcessError as e:
+        LOG.info('zfs list -o name,mountpoint '
+           'fails.\n (Error: %s)' % e)
+        return 1
```

Comment:
@tchaikov 
Removed the return. 
But if listing fails, there is not much more we can do?